test/controllers/user.js

```diff
@@ -13,130 +13,160 @@ const { copy } = require('../../app');
 
```

Comment:
the pull request says that the changes are for the language endpoint. Should this file to be in a different pull request??